### DISCUSSION TITLE
[Feature Request] Home "sort by" equal representation from all sublemmys

### DISCUSSION THREAD

**Discussion by jeff-hykin** [aeharding/voyager#604] [Ideas]
First, this app is fantastic, great job! One of the best PWA's I've seen in a long time.

The issue is; I will be subscribed to one sublemmy that posts all the time and it ends up completely dominating my home feed (drowning everything else out). Instead of showing posts chronologically it would be really nice to be able to grab one post per sublemmy.

E.g. sort posts within each sublemmy chronologically (or by any sort method) then pull one post from each sublemmy instead of putting all posts into an aggregate, then sorting.

**Comment by aeharding** [comment]
Hmm, interesting idea. This may be more appropriate for Lemmy to implement as a sorting option?

Turning into a discussion!

**Reply by jeff-hykin** [reply]
True! it would be nice to have it built into the core, maybe I'll open up and issue over there
